docs/integrations/common-workflow-steps/marathon-cloud.md

```diff
@@ -0,0 +1,59 @@
+
+
+### Prerequisites
+
+This component needs some prerequisites for proper operation; the necessary steps are given in the table below with details.
+
+:::caution
+Please note that this component works separately on both the **Android** and **iOS** platforms. The requirements for both platforms are different.
+:::
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Xcodebuild for Testing**](https://docs.appcircle.io/workflows/ios-specific-workflow-steps/xcodebuild-for-testing) | After the Xcodebuild for Testing step runs, the test IPA paths `$AC_TEST_IPA_PATH` and `$AC_UITESTS_RUNNER_PATH` will be created automatically. So that the **Marathon Cloud** component depends on these paths. |
+| [**Android Build for UI Testing**](https://docs.appcircle.io/workflows/android-specific-workflow-steps/android-build-for-ui-testing) | The Android Build for UI Testing step must be executed to obtain the necessary Android app outputs for processing. |
+
+#### For Android
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2990-marathonOrderAnd.png' />
+
+#### For iOS
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2990-marathonOrder.png' />
+
+
+### Input Variables
+
+The parameters required for the operation of this component are given below with explanations. 
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2990-marathonInput.png' />
+
+:::warning
+**Do not hard-code sensitive variables, such as tokens and API keys, directly to the parameters in the step.**
+
+We recommend using [**Environment Variables**](https://docs.appcircle.io/environment-variables/) groups for such sensitive variables.
+:::
+
+| Variable Name                            | Description                         | Status           |
+|-------------------------------|------------------------------------------------|------------------|
+| `$AC_MARATHON_API_KEY`        | Marathon Cloud API key. To get API key, please follow the Marathon Cloud [**documentations**](https://docs.marathonlabs.io/?_gl=1*fsz3tj*_ga*MTYyMzE3NDMwOC4xNzA2Njk3MzA5*_ga_7RE7PPY2QW*MTcxMTAwMzEyNC4yNS4xLjE3MTEwMDU5ODYuMjQuMC4w*_gcl_au*MjA1NTI4NzUyNC4xNzA2Njk3MzUw#api-key). | Required |
+| `$AC_MARATHON_TEST_NAME`      | This name will be shown on the Marathon Cloud dashboard test run list. | Required |
+| `$AC_MARATHON_APP_PATH`       | This path will be generated by relevant testing step(`Android Build for UI Testing` or `Xcodebuild for Testing`). | Required |
+| `$AC_MARATHON_UITEST_RUNNER_APP_PATH` | This path will be generated by relevant testing step(`Android Build for UI Testing` or `Xcodebuild for Testing`). | Required |
+| `$AC_OUTPUT_DIR`              | Specifies the path for Marathon Cloud outputs. | Required |
```

Comment:
Could you please specify that this will be automatically identified? so the user does not have to search for this value.